packages/flutter/lib/src/widgets/automatic_keep_alive.dart

```diff
@@ -144,7 +144,8 @@ class _AutomaticKeepAliveState extends State<AutomaticKeepAlive> {
   }
 
   VoidCallback _createCallback(Listenable handle) {
-    return () {
+    late final VoidCallback callback;
```

Comment:
```suggestion
    final VoidCallback callback;
```

Reply:
I needed to add the `late` because Dart doesn't know for sure that `callback` will be initialized when it is used within itself (`handle.removeListener(callback);`):

> The final variable 'callback' can't be read because it's potentially unassigned at this point.

Reply:
ahh, I see